ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -113,6 +118,17 @@
       [production_bod_scan]
       database-uri = mongodb://scan-reader:{{ mongo_users.other_users | selectattr("user", "eq", "scan-reader") | map(attribute="password") | first }}@database1:27017/scan
       database-name = scan
+  when: cyhy_conf_result.stat.exists == False
+
+- name: Check if desired additions are in cyhy.conf and if not append them
+  lineinfile:
+    dest: /etc/cyhy/cyhy.conf
+    line: '{{ item }}'
+  with_items:
+    - '[production_bod_scan]'
+    - database-uri = mongodb://scan-reader:{{ mongo_users.other_users | selectattr("user", "eq", "scan-reader") | map(attribute="password")| first }}@database1:27017/scan
+    - 'database-name = scan'
+
```

Comment:
I don't think this is a good way to handle this problem.  Why include the `production` section from the commander?

Better might be to create the `cyhy.conf` file that feeds needs if one does not already exist.  If one does exist then add the desired lines as you are doing.  You will want to add a  `when: cyhy_conf_result.stat.exists == True` to this block since there's no need for it to run if we're creating the `cyhy.conf` file from whole cloth.  Then there should be a parallel structure in the `cyhy_commander` Ansible code.

That way the correct file will get created regardless and feeds need not know anything about what commander wants in its `cyhy.conf` file.

Reply:
OK, I think I have a better solution in ee1697b.  Note that the feeds and commander Ansible roles now have a parallel structure.  It should not matter which one is run first, nor should it matter if they are run multiple times.

@dav3r and @KyleEvers, please let me know what you think.

Reply:
It seems I misunderstood what was going on.  feeds needs the commander/production `cyhy.conf` section.  This is corrected in commit 0dffd4a.